tests/e2e/env_spec.js

```diff
@@ -31,25 +31,25 @@ describe("Electron app environment", function () {
 		return helpers.stopApplication(app);
 	});
 
-	it("should open a browserwindow", function () {
+	it("should open a browserwindow", async function () {
+		await app.client.waitUntilWindowLoaded();
+		app.browserWindow.focus();
+		const cnt = await app.client.getWindowCount();
+		const min = await app.browserWindow.isMinimized();
+		const dev = await app.browserWindow.isDevToolsOpened();
+		const vis = await app.browserWindow.isVisible();
+		const foc = await app.browserWindow.isFocused();
+		const bounds = await app.browserWindow.getBounds();
+		const title = await app.browserWindow.getTitle();
 		return (
-			app.client
```

Comment:
This return is no longer necessary as it doesn't return a promise anymore. I think it can be simplified and easier to read by adding one assertion per row.

e.g.
```js
expect(cnt).to.equal(1);
expect(min).to.be.false;
```